### ISSUE TITLE
libkaldi_jni.so support for 32-bit architecture?

### ISSUE THREAD

**Issue by ammyt** [alphacep/kaldi-android-demo#53]
I have noticed that the app would immediately crash on x86 devices, This is probably because there is no libkaldi_jni.so for x86.

Can this be added?

**Comment by nshmyrev** [comment]
No much interest in that to be honest. Any reason you have to support it? Most devices are 64-bit already.

**Comment by ammyt** [comment]
Yes, I need to deploy the app I integrated vosk into to our devices in our domain, which are 32-bit Intel Atom devices. I really haven't found any alternative that works as well as this that's why it would be great if a 32 bit library could be provided

**Comment by ammyt** [comment]
I have found this compilation tutorial
https://medium.com/swlh/compile-kaldi-for-64-bit-android-on-ubuntu-18-70967eb3a308
But I don't know how relevant it is, or if it is possible to adapt this for 32 bit
